packages/blade/src/components/SkipNav/SkipNav.stories.tsx

```diff
@@ -0,0 +1,84 @@
+import type { ComponentStory, Meta } from '@storybook/react';
+import {
+  Title as StorybookTitle,
+  Subtitle,
+  Primary,
+  ArgsTable,
+  Stories,
+  PRIMARY_STORY,
+} from '@storybook/addon-docs';
+import { Highlight } from '@storybook/design-system';
+import type { ReactElement } from 'react';
+import { Text } from '../Typography';
+import { SkipNavContent, SkipNavLink } from '.';
+
+const Page = (): ReactElement => {
+  return (
+    <>
+      <StorybookTitle />
+      <Subtitle>
+        The SkipNav component lets users skip the navigation and jump to the main content of the
```

Comment:
Can we mention here that this is only available for web?

Reply:
Yes